selfservice/strategy/oidc/types.go

```diff
@@ -40,8 +46,10 @@ func NewCredentials(provider, subject string) (*identity.Credentials, error) {
 }
 
 type ProviderCredentialsConfig struct {
-	Subject  string `json:"subject"`
-	Provider string `json:"provider"`
+	Subject               string `json:"subject"`
+	Provider              string `json:"provider"`
+	EncryptedAccessToken  string `json:"encrypted_access_token"`
```

Comment:
Since these can now be encrypted and unencrypted let’s remove the encrypted name and name them initial as we do not update these :)

```suggestion
	InitialAccessToken  string `json:"initial_access_token"`
```